templates/notes/getting_started.md

```diff
@@ -88,7 +88,7 @@ $$$
 
 ###1.5. Images
 
-Images can be dragged and dropped over the editor. Alternatively, you can use the standard markdown syntax to insert images.
+Images can be dragged and dropped over the editor. Alternatively, you can copy&paste image data (not files) directly to editor (supported only by Chrome, Firefox and Internet Explorer 11). Lastly, you can use the standard markdown syntax to insert images.
```

Comment:
I'll have a look when I pull the code. I'd expect two things, as a user:

* The ability to paste a screenshot (Windows saves them to clipboard, right?)
* The ability to paste an image file (OS X saves them as files on the desktop)

The second point is not nearly as important, since you can still drag files around. However, the keyboard shortcut must not cause errors in any case. Essentially, it should mimic the behaviour of pasting images in Google Docs. However, any sort of pasting is better than nothing.